bindings/generator/templates/binding_electron_meths.rs.j2

```diff
@@ -259,6 +265,7 @@ match {{ rs_value }} {
 }
 {%- elif type.kind == "none" -%}
 {
+    #[allow(clippy::let_unit_value)]
```

Comment:
Missing fixme comment ?

Reply:
There is no fix possible, or litterally a big change in jinja, we have to test before if the result is `None`